app/front/scripts/components/babbage-geoview/utils.js

```diff
@@ -121,11 +164,34 @@ function prepareGeoJson(geoJson, options) {
   updateValues(geoJson, options);
 }
 
-function formatValue(value) {
+function formatValue(value, currencySign) {
   if (_.isUndefined(value)) {
     return 'N/A';
   }
-  return value;
+  return formatAmount(value, currencySign);
+}
+
+function formatAmount(value, currencySign) {
+  var suffixes = [
+    [1000000000, ' Milliards'],
```

Comment:
I think it should be 'Billions'

(https://en.wikipedia.org/wiki/Long_and_short_scales)